packages/cdk/lib/cloudwatch-logs-management-props.ts

```diff
@@ -0,0 +1,7 @@
+import type {GuStackProps} from "@guardian/cdk/lib/constructs/core";
+
+export interface CloudwatchLogsManagementProps
```

Comment:
I think we can create an interface per stack, `retentionInDays` is only used by `CloudwatchLogsRetention`, and `logShippingPrefixes` by `CloudwatchLogsManagement`.

This can be done in a follow-up PR though, as it'll likely mean the contents of `bin/cdk.ts` will change a bit too.